src/cartridges/int_adyen_SFRA/cartridge/client/default/js/adyen_checkout/renderGenericComponent.js

```diff
@@ -138,10 +138,5 @@ module.exports.renderGenericComponent = async function renderGenericComponent()
     );
     firstPaymentMethod.checked = true;
     helpers.displaySelectedMethod(firstPaymentMethod.value);
-
-    if (store.checkoutConfiguration.paymentMethodsConfiguration?.amazonpay) {
-      store.checkoutConfiguration.paymentMethodsConfiguration.amazonpay.amount =
-        data.amount;
```

Comment:
We don't need the `amount` anymore in the config?

Reply:
the amount we add to the generic checkoutConfiguration object actually worked so this is not needed anymore